test/test_enclave_wrangler.py

```diff
@@ -6,37 +6,125 @@
+    def test_upload(self):
+        # todo: when data format is complete, copy over file(s) from termhub/csets/n3c-upload-jobs to TEST_INPUT_DIR
+        csv_dir = os.path.join(PROJECT_ROOT, 'termhub-csets', 'n3c-upload-jobs')
+        # fname = os.path.join(csv_dir, 'Other Diabetes.csv')
+        fname = os.path.join(csv_dir, 'diabetes-recommended-csets-modifications', 'type-2-diabetes-mellitus.csv')
+
+        df = pd.read_csv(fname).fillna('')
+        omop_concepts = df[[
+            'concept_id',
+            'includeDescendants',
+            'isExcluded',
+            'includeMapped',
+            'annotation']].to_dict(orient='records')
+        new_version = {
+            "omop_concepts": omop_concepts,
+            "provenance": "Created through TermHub.",
+            "concept_set_name": "[DM]Type2 Diabetes Mellitus",
+            "limitations": "",
+            "intention": "",
+            "on_behalf_of": os.getenv('ON_BEHALF_OF')
+        }
+        try:
+            responses: Dict[str, Union[Response, List[Response]]] = upload_new_cset_version_with_concepts(**new_version)
+            for response in responses.values():
+                if isinstance(response, list):  # List[Response] returned by concepts upload
+                    for response_i in response:
+                        self.assertLess(response_i.status_code, 400)
+                else:
+                    self.assertLess(response.status_code, 400)
+        except Exception as err:
+            print(err, file=sys.stderr)
+
+        # Teardown
+        # todo: can remove try/except when properly implement teardown
+        #      {
+        #       "apiName": "delete-omop-concept-set-version",
+        #       "description": "Do not forget to the flag 'Is Most Recent Version' of the previous Concept Set Version to True!",
+        #       "rid": "ri.actions.main.action-type.93b82f88-bd55-4daf-a0f9-f6537bf2bce1",
+        #       "parameters": {
+        #         "omop-concept-set": {
+        #           "description": "",
+        #           "baseType": "OntologyObject"
+        #         },
+        #         "expression-items": {
+        #           "description": "",
+        #           "baseType": "Array<OntologyObject>"
+        #         }
+        #       }
+        #     },
+        # TODO: I think I need to do what it suggested above regarding 'most recent version'. I think this means that my
+        #  teardown step here will involve another step to update the previous version and label it most recent.
+        # todo: ask amin?: Why do i need to declare expression items?
+        # todo: how do I get the version? is the version returned in the response json?
+        # todo: where do I pass the version? omop-concept-set param? Ask amin, is that just poorly named? if that param
+        #  is indeed for the container, then is there not a way to stipulate a version? does it just delete the most
+        #  recent version?
+        # make_request(
```

Comment:
@Sigfried Unfortunately, I couldn't complete the teardown for this test as planned, because of some or all of these issues:
- [ ] 1. TODO: I think I need to do what it suggested above regarding 'most recent version'. I think this means that my teardown step here will involve another step to update the previous version and label it most recent.
- [ ] 2. todo: ask amin?: Why do i need to declare expression items?
- [ ] 3. todo: how do I get the version? is the version returned in the response json?
- [ ] 4. todo: where do I pass the version? omop-concept-set param? Ask amin, is that just poorly named? if that param is indeed for the container, then is there not a way to stipulate a version? does it just delete the most